dashboard/app/models/script.rb

```diff
@@ -241,6 +241,7 @@ def self.add_script(options, data)
         end
 #      end
 
+        # TODO use Level.find_by_key
```

Comment:
yeah, I was hoping it would be easier to use find_by_key in the script seed (since I took the key format from the .script file key format), but there's a lot going on in there (ie more than just loading the level...).